selenium/build-dev.sh

```diff
@@ -38,13 +42,31 @@ if [ "$browser" == "chrome/local" -o "$browser" == "opera/blink/local" -o "$brow
     cp $debWildcard "$dir_name"
     docker rm -f static-server || true
     docker run -d -p 8080:8043 -v "$dir_name":/srv/http --name static-server pierrezemb/gostatic
-    host_ip=$(ifconfig docker0 | grep inet | grep -v inet6 | awk '{print $2;}' | sed -e 's|addr:||g')
+    host_ip=$(ifconfig | grep -E "([0-9]{1,3}\.){3}[0-9]{1,3}" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1)
```

Comment:
Could we take this approach if `docker0` is not found?

Reply:
Fixed